web/modules/custom/mymodule/mymodule.module

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+*@file
+*This file implements how to implement the hoook function.
+*/
+
+/**
+*Implements hook_cron
+*/
+function mymodule_cron() {
+  \Drupal::messenger()->addMessage(t("Hello World from mymodule  hook_cron"));
+}
+
+/**
+* This implements hook_form_alter().
+*/
+function mymodule_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
```

Comment:
Please check in Drupal core how the form parameters for form_alter are mentioned.

Reply:
Done